core/lib/workarea/ext/jbuilder/jbuilder_cache.rb

```diff
@@ -0,0 +1,31 @@
+require 'jbuilder/jbuilder_template'
```

Comment:
Was this necessary? I'd think the jbuilder require in core.rb would have taken care of it.

Reply:
Whoops, yeah I think I was trying something out here.